packages/core/docs/PRIVACY.md

```diff
@@ -0,0 +1,7 @@
+Table of contents
+
+- [Cryptography Explainer](https://github.com/hoprnet/hoprnet/blob/Privacy-documentation/packages/core/docs/cryptography/Cryptography%20Explainer.md)
+- [Anonymity metrics](https://github.com/hoprnet/hoprnet/issues/2209)
```

Comment:
this could become another document

Reply:
You mean the anonymity metrics?

Reply:
I meant transforming the GH issue into a markdown file and adding the results to the repository